.pre-commit-config.yaml

```diff
@@ -1,6 +1,6 @@
 repos:
 -   repo: https://github.com/phantomcyber/dev-cicd-tools
-    rev: v1.4
+    rev: v1.6
```

Comment:
this should be `v1.7` after the release you created earlier today (well `pre-commit autoupdate` should pick up the new version) :)